tox.ini

```diff
@@ -8,4 +8,4 @@ deps =
     -rrequirements.txt
     pytest
 passenv = LABELBOX_TEST_API_KEY_PROD LABELBOX_TEST_API_KEY_STAGING LABELBOX_TEST_ENVIRON
-commands = pytest {posargs}
+commands = pytest -m "not slow" {posargs}
```

Comment:
we never run slow tests?

Reply:
It's a default in tox, if you just run `pytest` locally it will run them... So, not really sure when to run and when not :/ Wdyt?

Reply:
Is there a way to trigger build when the PR is approved? Then we could run all tests, including the slow ones?

Reply:
Well, there is: https://github.community/t/feature-request-trigger-action-on-pull-request-approved/18413/2

But the logic should then be: if (push and was_approved_before) or (is_approved), which contains different GH action events and I don't want to get into that right now. Will first just remove the "not slow" flag and have everything run.